wiki/architecture_decisions.md

```
@@ -12,7 +12,7 @@ Architecture Decision Records help us:
 
 ## ADR Index
 
-### [ADR-001: Enhanced Handler System Implementation](adr-001-enhanced-handler-system.md)
+### [ADR-001: Enhanced Handler System Implementation](adr_001_enhanced_handler_system.md)
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)